tests/test.ts

```diff
@@ -0,0 +1,175 @@
+import * as anchor from "@coral-xyz/anchor";
```

Comment:
OK to leave for now, PR https://github.com/wen-community/wen-new-standard/pull/34 looks like it covers testing in a more scalable way so this may be redundant.